internal/core/metadata/rest_device.go

```diff
@@ -320,18 +320,17 @@ func restGetDeviceById(
 	errorHandler errorconcept.ErrorHandler) {
 
 	vars := mux.Vars(r)
-	var did string = vars[ID]
+	var did = vars[ID]
 
 	res, err := dbClient.GetDeviceById(did)
 	if err != nil {
 		errorHandler.HandleOneVariant(w, err, errorconcept.Database.NotFound, errorconcept.Default.BadRequest)
 		return
 	}
 	w.Header().Set(clients.ContentType, clients.ContentTypeJSON)
-	json.NewEncoder(w).Encode(res)
+	_ = json.NewEncoder(w).Encode(res)
 }
 
-//Shouldn't need "rest" in any of these methods. Adding it here for consistency right now.
```

Comment:
From our offline conversation:

> Your frustration is palpable.  Posting here instead of in the issue in an attempt to deescalate.  You updated the comment.  You changed it.  Own the change.  Yes, it was only to add a space.  Recognize the comment isn't helpful and remove it.   Do that for whatever unhelpful comments you've touched.  If they are docblocks and you removed them, replace them with a proper docblock comment.  I'm not asking you to update everything (or to swing at windmills).  I am asking you to not ignore a trash comment and to leave the code in a better state than you found it in.

The expectation here is that you were going to delete this comment (which you did) and because it's a docblock comment for restCheckForDevice that you'd replace it with a proper docblock comment that describes what the function does.